mongo/src/test/java/org/axonframework/extensions/mongo/eventsourcing/eventstore/MongoOptionsFactoryTest.java

```diff
@@ -54,13 +58,16 @@ void testCreateMongoOptions_customSet() {
         factory.setConnectionTimeout(11);
         factory.setMaxWaitTime(3);
         factory.setSocketTimeOut(23);
-        factory.setThreadsAllowedToBlockForConnectionMultiplier(31);
+//        factory.setThreadsAllowedToBlockForConnectionMultiplier(31);
```

Comment:
We can drop this assertion as the `setThreadsAllowedToBlockForConnectionMultiplier` logic will be removed too.